recipe/meta.yaml

```diff
@@ -58,7 +59,7 @@ test:
     - rpy2.robjects  # [not win]
     - rpy2.robjects.lib  # [not win]
   commands:
-    - pytest -s -rxs -v rpy2/tests -k "not test_display_args"  # [not win]
+    - pytest -s -rxs -v rpy2/tests -k "not test_display_args and not test_svg_plotting_args"  # [not win]
```

Comment:
```suggestion
    - pytest -s -rxs -v rpy2/tests -k "not test_display_args and not test_svg_plotting_args"  # [not (win or aarch64 or ppc64le)]
```

Reply:
That is maybe better than disabling two many tests for all archs.

Reply:
Agreed. On top of these I added tests for aarch64 and ppc64le that just skip the offending tests.